can-stache.js

```diff
@@ -563,6 +563,12 @@ stache.from = mustacheCore.getTemplateById = function(id){
 		var el = DOCUMENT().getElementById(id);
 		if(el) {
 			templates[id] = stache("#" + id, el.innerHTML);
+		} else {
+			//!steal-remove-start
+			if (process.env.NODE_ENV !== 'production') {
```

Comment:
This doesn't seem like the correct place to put this warning. This will cause the same warning if you do `stache.from("foo")` and there is no `<script type='text/stache' id='foo'>` on the page. I think `Unable to find partial 'foo'` will be pretty confusing in this case since `stache.from` isn't about partials.

I think the correct fix is inside `makeLiveBindingPartialRenderer`, somewhere in here:

https://github.com/canjs/can-stache/blob/bd91e957535d4aea80b31ac825ba95795b8d1df7/src/mustache_core.js#L223-L230

This would also allow us to add the filename and lineNumber in the warning, which would make it much easier to track down what you're doing wrong.

Reply:
@phillipskevin Thank you, I changed the fix as you reviewed